src/pages/documentation/media-for-edge-network/index.md

```diff
@@ -0,0 +1,78 @@
+import Tabs from './tabs/index.md'
+
+# Adobe Edge Media Analytics for Audio and Video
+
+<InlineAlert variant="info" slots="text"/>
+
+This extension requires the setting up of Datastreams with Media forwarding enabled, Dataset, XDM Schema and other AEP settings. Please follow this `backendDocumentationLink` before configuring and implementing the SDK.  
```

Comment:
Can you change this into a "Before starting" section like we have for Edge https://developer.adobe.com/client-sdks/documentation/edge-network/#before-starting
Here you can state that the Media for Edge Network requires the Edge and Edge Identity extensions, and the XDM Schema must include the MediaCollection field group and the Datastream must be configured with Media forwarding enabled. Follow these `backendDocumentationLink` instructions before configuring and implementing the SDK.